db/migrate/20210830152920_add_coordinates_to_property.rb

```diff
@@ -0,0 +1,6 @@
+class AddCoordinatesToProperty < ActiveRecord::Migration[6.0]
+  def change
+    add_column :properties, :latitude, :float
+    add_column :properties, :longtitude, :float
```

Comment:
small typo here: should be `longitude` rather than `longtitude`.

- I'm not sure if I can change it here as its a migration file?